code/game/machinery/computer/security.dm

```diff
@@ -42,6 +44,18 @@
 	add_fingerprint(user)
 	ui_interact(user)
 
+/obj/machinery/computer/secure_data/proc/createlog(toprint)
+	if(isnull(toprint))
+		return 0
+	else
+		var/obj/item/weapon/paper/P = toprint
+		playsound(loc, "sound/goonstation/machines/printer_dotmatrix.ogg", 50, 1)
+		to_chat(usr, "<span class='notice'>Printing file [P.name].</span>")
+		sleep(50)

```

Comment:
unless you specifically intend to halt a process, you should be using `addtimer`.

`spawn` is acceptable to be used if you need to spawn of a thread instantly, but sleeping as it's used here? Shouldn't be doing this.